calico_containers/pycalico/netns.py

```diff
@@ -209,6 +238,22 @@ def remove_ip_from_ns_veth(namespace, ip, veth_name_ns):
                        "dev", veth_name_ns])
 
 
+class Route(object):
+    def __init__(self, route_output):
+        self.route_output = route_output
+        self.default = route_output.startswith("default")
+        match = re.search('metric\s+(\d+)', route_output)
+        self.metric = int(match.group(1)) if match else 0
+
+    def with_incremented_metric(self):
+        """
+        Returns a string representing the route with its metric incremented
+        by 1. 
+        """
+        route_without_metric = re.sub('metric\s+\d+', '', self.route_output)
+        return "{} metric {}".format(route_without_metric, self.metric + 1)
```

Comment:
Think it's worth handling the maximum metric case (4294967295)?  E.g. `max(self.metric+1, MAX_METRIC)`